README.md

```diff
@@ -617,7 +617,7 @@ Payment settlement can be triggered by any rail participant:
 // Settlement call - can be made by client, service provider, or operator
 (uint256 amount, uint256 settledEpoch, string memory note) = Payments(paymentsContractAddress).settleRail(
     railId,        // Rail ID
-    block.number   // Settle up to current epoch
+    block.timestamp   // Settle up to now
```

Comment:
Specifying a current timestamp parameter might be harder than specifying a block number. Two solutions to this:
1. `min(block.timestamp, untilTime)`
2. add helper to settle up to now.

Reply:
> 1. `min(block.timestamp, untilTime)`
> 2. add helper to settle up to now.


For UX, I think I want to do one of these whether or not we make this change.